pom.xml

```diff
@@ -800,7 +800,8 @@
         <dependency>
           <groupId>org.graalvm.buildtools</groupId>
           <artifactId>junit-platform-native</artifactId>
-          <version>0.9.6</version>
+          <!-- Using older version; upgrade to latest once issues are resolved. -->
```

Comment:
Do you want to add a link to the issue? Some link that shows error message or stacktrace would be nice.